### ISSUE TITLE
Error message while switching state

### ISSUE THREAD

**Issue by Raykesjeng** [F4stFr3ddy/homebridge-jablotron-alarm#37]
I've connected my Jablotron account and the config file has provided all the ID's, but when I try to switch the state I get this error:
[2019-10-28 22:58:54] [Jablotron] Switching section section_2 (using section_2) to new state: set
[2019-10-28 22:58:57] [Jablotron] {
  status: false,
  error_status: 'bypass_dialog_timed',
  checksum: '9f785d19e3d329edfb6ebff755a44f79',
  server_id: '8731203-1572299937',
  client_id: null
}
[2019-10-28 22:58:57] [Jablotron] Error: bypass_dialog_timed
    at C:\Users\Gebruiker\AppData\Roaming\npm\node_modules\homebridge-jablotron\lib\jablotron-client.js:25:31
    at IncomingMessage.<anonymous> (C:\Users\Gebruiker\AppData\Roaming\npm\node_modules\homebridge-jablotron\lib\jablotron-client.js:56:17)
    at IncomingMessage.emit (events.js:215:7)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)

Running nodejs on Windows 10 machine.

**Comment by F4stFr3ddy** [comment]
Hi,

Are you sure the account has been activated? Can you use this account via the app / web and it works fine?

Best regards,

Freddy

**Comment by Raykesjeng** [comment]
Hi Fred

I investigated a little further and I think it has to do with a warning
message I get before I can activate a section. I don’t have a gsm card
installed and get a warning. Disabling of the alarm works fine. Is there a
way around the warning message?

Raymond

Op di 29 okt. 2019 om 19:22 schreef Fred de Gier <notifications@github.com>

**Comment by F4stFr3ddy** [comment]
Interesting, what kind of message do you get? I have never had this warning so its going to be a bit hard to debug for me.

**Comment by Raykesjeng** [comment]
Hi Fred,

Thanks for thinking with me. I found a workaround by changing the way
sections get activated. This is my error message on the panel and on the
website:
[image: image.png]
[image: image.png]

I now changed this option in F-Link:

[image: image.png]
For people not understanding Dutch, its the option to enable without
warnings. After setting this to Always, I don't get a warning anymore and
the alarm can be activated through home bridge.

Regards,
Raymond
